### ISSUE TITLE
Hard to distinguish warnings and errors

### ISSUE THREAD

**Issue by ad-si** [purescript/purescript#3778]
At the moment it's really hard to visually distinguish warnings from errors:

![image](https://user-images.githubusercontent.com/36796532/73218896-ce26e280-4152-11ea-8c1a-07e6cd886fd1.png)

It would be cool if the difference was more prominent. E.g. through using different colors and adding ASCII art for sectioning the output.

Maybe something like this (pseudocode for coloring):

```
<yellow>=== Warning 50 of 50:==================================================</yellow> 

  at src/Neodoc/Evaluate/Reduce.purs:97:16 - 97:17 (line 97, column 16 - line 97, column 17)

…

<red>=== Error 1 of 5===========================================================</red>

  at src/Neodoc/ArgParser/Parser.purs:685:1 - 740:48 (line 685, column 1 - line 740, column 48)

    An anonymous function argument appears in an invalid context.

…
```

What do you think?

**Comment by natefaubion** [comment]
You might try `purescript-psa` for console based output.

**Comment by ad-si** [comment]
I'm seeing this as output from `spago build`.  Can I use psa with spago?
But this issues is about improving the default anyways :wink:

**Comment by natefaubion** [comment]
https://github.com/spacchetti/spago/issues/252

**Comment by hdgarrood** [comment]
psa is a good option for now, but I think this is worthy of considering; I'm also keen on improving the defaults. I'd be up for folding basically everything that psa does back into the main compiler. See also: #1898, #2308.

**Comment by natefaubion** [comment]
Yes, as the author of `psa`, I too would prefer it was just a compiler default.

**Comment by ad-si** [comment]
Look at this, psa does already seem to be the default for spago 😳:
```txt
$ spago --help
…
  -P,--no-psa              Don't build with `psa`, but use `purs`
…
```
But it doesn't seem to be used 🤔

**Comment by natefaubion** [comment]
Maybe it needs to be globally installed?

**Comment by hdgarrood** [comment]
I think it needs to be on your PATH.

**Comment by ad-si** [comment]
Ah yes, either spago and purescript-psa are both installed locally or both globally, but the global spago doesn't pick up a local purescript-psa. Should maybe be implemented. Tools like webpack use the local version if they find one iirc.

**Comment by winston0410** [comment]
For anyone reading this, if you need to use spago with psa, simply install both globally. They will work together automatically.
